# HG changeset patch
# User Rik <rik@octave.org>
# Date 1513011072 28800
#      Mon Dec 11 08:51:12 2017 -0800
# Node ID f997aa4be0ceb4d06a0908c289d91edd59eeb050
# Parent  a7bc68ce08f19792d7f5233b4b382cc1b97ac79a
get_description.m: Fix quote character error introduced in b1d1229d9e83 (bug #52639).

* get_description.m: Close quote started with ' with another '.

diff --git a/scripts/pkg/private/get_description.m b/scripts/pkg/private/get_description.m
--- a/scripts/pkg/private/get_description.m
+++ b/scripts/pkg/private/get_description.m
@@ -51,17 +51,17 @@ function desc = get_description (filenam
         keyword = tolower (strtrim (line(1:colon-1)));
         value = strtrim (line (colon+1:end));
         if (length (value) == 0)
             fclose (fid);
             error ("The keyword '%s' of the package '%s' has an empty value",
                     keyword, desc.name);
         endif
         if (isfield (desc, keyword))
-          warning ('pkg: duplicate keyword "%s" in DESCRIPTION, ignoring",
+          warning ('pkg: duplicate keyword "%s" in DESCRIPTION, ignoring',
                    keyword);
         else
           desc.(keyword) = value;
         endif
       endif
     endif
     line = fgetl (fid);
   endwhile
